client/my-sites/stats/feedback/index.tsx

```diff
@@ -10,8 +15,37 @@ const FEEDBACK_ACTION_LEAVE_REVIEW = 'feedback-action-leave-review';
 const FEEDBACK_ACTION_SEND_FEEDBACK = 'feedback-action-send-feedback';
 const FEEDBACK_ACTION_DISMISS_FLOATING_PANEL = 'feedback-action-dismiss-floating-panel';
 
+const FEEDBACK_PANEL_PRESENTATION_DELAY = 3000;
 const FEEDBACK_LEAVE_REVIEW_URL = 'https://wordpress.org/support/plugin/jetpack/reviews/';
 
+const FEEDBACK_SHOULD_SHOW_PANEL_API_KEY = NOTICES_KEY_SHOW_FLOATING_USER_FEEDBACK_PANEL;
+const FEEDBACK_SHOULD_SHOW_PANEL_API_HIBERNATION_DELAY = 30; // 30 seconds for now
+// Examples values:
+// 30 minutes = 60 * 30;
+// 30 days = 3600 * 24 * 30;
+
+function useNoticeVisibilityHooks( siteId: number ) {
+	const { data: shouldShowFeedbackPanel, refetch } = useNoticeVisibilityQuery(
+		siteId,
+		FEEDBACK_SHOULD_SHOW_PANEL_API_KEY
+	);
+
+	const { mutateAsync } = useNoticeVisibilityMutation(
+		siteId,
+		FEEDBACK_SHOULD_SHOW_PANEL_API_KEY,
+		'postponed',
+		FEEDBACK_SHOULD_SHOW_PANEL_API_HIBERNATION_DELAY
+	);
+
+	const updateFeedbackPanelHibernationDelay = () => {
+		mutateAsync().then( () => {
+			refetch();
+		} );
+	};
+
+	return { shouldShowFeedbackPanel, updateFeedbackPanelHibernationDelay };
```

Comment:
I suspect we should test the loading status because the default value of the notice is set to true.

Reply:
Addressed in https://github.com/Automattic/wp-calypso/pull/94479